### ISSUE TITLE
Placeholder: TOP-10 challenges in island leader

### ISSUE THREAD

**Issue by Incluuu** [BentoBoxWorld/Challenges#333]
If levels has top island level. I need Challenges placeholders for quest TOP
![image](https://github.com/BentoBoxWorld/Challenges/assets/67196311/aaa581f8-40b8-454b-a347-45a664bb6752)

**Comment by tastybento** [comment]
Do you mean some kind of top ten for the most challenges done? That's an interesting idea. It doesn't exist right now though.

**Comment by BONNe** [comment]
The issue is that I kinda do not like idea to have a top handling in each addon. That sounds kinda lot of same process in multiple add-ons. 
I would prefer if there would be a generic top addon that is fed with the data, like leader heads.

**Comment by Incluuu** [comment]
> Do you mean some kind of top ten for the most challenges done? That's an interesting idea. It doesn't exist right now though.

yes, since now all players added to the island do not add up, I would like only the leader to be displayed from the islands top challenges

**Comment by Incluuu** [comment]
Do you already have any updates regarding this matter? When can we expect this option to be added?

**Comment by tastybento** [comment]
I can start to work through the open issues in Challenges. I'm just tidying up Limits.

**Comment by tastybento** [comment]
Reasonable ask — Level has top-ten placeholders, Challenges has per-player counts but no rankings.

What's needed: a method on `ChallengesManager` returning the top N players by completed-challenge count for a world (the data is already in `ChallengesPlayerData`), cached with periodic refresh so placeholder lookups don't scan every player record, plus placeholders like `challenges_top_completed_name_1..10` / `challenges_top_completed_count_1..10` registered per gamemode.

The caching is the part that needs care (player data is loaded lazily), which is why this isn't a quick one. Marking P2 — a nice leaderboard feature, no stability impact.
